### PR TITLE
Adjusted calendar/titleReplace, customEvents for MM² PR #3942

### PR DESCRIPTION
**Pull request by kaennchenstruggle**

### CODE REVIEW

**Comment by rejas** [comment]
Thanks, but can you base it against the develop branch since this is feature will go live only when the next mm2 release is happening?

**Comment by kaennchenstruggle** [comment]
hope this is okay now :)

File: modules/calendar.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The following properties can be configured:
...
...br>`customEvents: [{keyword: 'Geburtstag', symbol: 'birthday-cake', color: 'Gold', transform: { search: '^([^\']*) \'(\\d{4})$' , replace: '$1 ($2.)', yearmatchgroup: 2}},{keyword: 'in Hamburg', transform: { search: ' in Hamburg$' , replace: ''}} ]`<br>Will transform<br>![before](https://user-images.githubusercontent.com/54073894/278068297-67ca65f4-24c3-46a8-bee8-0519e4bba3f5.png)<br>to<br>![after](https://user-images.githubusercontent.com/54073894/278068458-0b4af07d-d3d9-4644-a4a6-e8c402598208.png)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```
Comment (rejas) [inline modules/calendar.md:66]: just one more thing: put images into the `modules/screenshot` folder

**Review by rejas — CHANGES_REQUESTED** [review]

File: modules/calendar.md (hunk 1 above)
Comment (kaennchenstruggle) [reply modules/calendar.md]: done :)

**Review by rejas — APPROVED** [review]

**Pull request merged**